lib/Braintree.php

```diff
@@ -139,8 +137,8 @@
 require_once('Braintree/Exception/TestOperationPerformedInProduction.php');
 require_once('Braintree/Test/Transaction.php');
 
-if (version_compare(PHP_VERSION, '5.4.0', '<')) {
-    throw new Braintree_Exception('PHP version >= 5.4.0 required');
+if (version_compare(PHP_VERSION, '5.3.0', '<')) {
+    throw new Braintree_Exception('PHP version >= 5.3.0 required');
```

Comment:
Is it worth explaining why/how we're packaging the official Braintree library while changing their version check - perhaps in the release notes?